### ISSUE TITLE
CW - Look up delivery address

### ISSUE THREAD

**Issue by MiklosGyalog** [MiklosGyalog/hello-world#3]
### User story -> CW - Look up delivery address ###
As an End user I want to add my address to the delivery order without having to input all the details manually, in order to make the process more convenient and fast.

**Out of scope of this user story**: PUDO destination selections. See separate user story.

### Acceptance criteria ###
* The user is presented with the interface to capture address details.
* The user cannot proceed to any subsequent screen unless address is provided.
* Given that the user selected delivery, when inputting the delivery address, the following details should be captured.
   * The user can manually type in address details or allow the browser to fill in the form.
   * The user can input a post code and trigger an address lookup.

|	Field	|	Mandatory / optional	|	Acceptance criteria	|	Notes	|
|	---	|	---	|	---	|	---	|
|	Full name	|	M	|	If available, the full name is pre-filled from previously captured information (e.g. logged in user, cookie from previous session, etc.)	|		|
|	Address Line 1 or company name	|	M	|		|		|
|	Address Line 2	|	O	|	Auto-fill when a valid UK postcode is provided.	|		|
|	Address Line 3	|	O	|	Auto-fill when a valid UK postcode is provided.	|		|
|	Town/City	|	M	|	Auto-fill when a valid UK postcode is provided.	|		|
|	County (optional)	|	O	|		|	<MG> Is this required at all?	|
|	Country	|	M	|	Auto-fill when a valid UK postcode is provided.	|		|
|	Post code	|	M	|	When provided, the user can choose to look up the remaining values of the address.	|	<MG> What should trigger lookup; should there be a button to initiate, or should it be performed automatically upon Enter or click-away?	|
|	Phone number	|	M	|	Validate if it complies with UK phone number syntax.	|	 International dial codes.	|
|		|		|	Valid numbers are accepted.	|		|
|		|		|	Invalid numbers are not accepted. The user is required to update the phone number either to a valid UK number, or to include international dial code.	|		|
